using tar in that way to copy files changes usr/include producing a share
with 00-dirtree
Alejandro Mery

--- ./Makefile.orig	2005-01-04 11:22:18.000000000 -0300
+++ ./Makefile	2005-01-04 11:22:48.000000000 -0300
@@ -191,10 +191,8 @@
 
 # Installs header files and development library links.
 install_dev:
-	$(INSTALL) -d $(PREFIX)$(DEVEL_PREFIX)lib
-	$(INSTALL) -d $(PREFIX)$(DEVEL_PREFIX)include
 	-$(INSTALL) -m 644 lib/*.[ao] $(PREFIX)$(DEVEL_PREFIX)lib/
-	tar -chf - include | tar -xf - -C $(PREFIX)$(DEVEL_PREFIX);
+	tar -chf - include/* | tar -xf - -C $(PREFIX)$(DEVEL_PREFIX);
 ifneq ($(strip $(UCLIBC_HAS_FLOATS)),y)
 	# Remove floating point related headers since float support is disabled.
 	$(RM) $(PREFIX)$(DEVEL_PREFIX)include/complex.h
